### PR TITLE
Delete buggy selected_rows functor

### PR DESCRIPTION
**Pull request by sneaxiy**
`Add` and `Mul` functors for `SelectedRows` are wrong. Delete them.

### CODE REVIEW

**Review by chengduoZH — APPROVED** [review]
Cool

File: paddle/fluid/operators/math/selected_rows_functor.h [hunk 1]
```diff
@@ -88,57 +88,6 @@ struct MergeAdd {
...
-    out.mutable_value()->mutable_data<T>(input1.value().dims(),
-                                         context.GetPlace());
-    auto e_out = framework::EigenVector<T>::Flatten(*(out.mutable_value()));
-    auto e_in1 = framework::EigenVector<T>::Flatten(input1.value());
-    e_out.device(*context.eigen_device()) = input2 * e_in1;
-    return out;
-  }
-};
-
 enum class ScatterOps { ASSIGN, ADD, SUB, SUBBY, MUL, DIV, DIVBY };
```
Comment (typhoonzero) [inline paddle/fluid/operators/math/selected_rows_functor.h]: Need update this enum

**Comment by typhoonzero** [comment]
Can delete `UpdateToTensor` too, it's not used.

**Comment by sneaxiy** [comment]
Ok, I would delete them in the next PR. @typhoonzero

**Pull request merged**